css/properties/box-decoration-break.json

```diff
@@ -0,0 +1,78 @@
+{
+  "css": {
+    "properties": {
+      "box-decoration-break": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/box-decoration-break",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "This property is only supported for inline elements."
+            },
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "22",
+              "notes": "This property is only supported for inline elements."
+            },
+            "chrome_android": {
+              "prefix": "-webkit-",
+              "version_added": "4.4",
+              "notes": "This property is only supported for inline elements."
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "32"
+            },
+            "firefox_android": {
+              "version_added": "32"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": [
+              {
+                "partial_implementation": true,
+                "version_added": "15"
+              },
+              {
+                "version_added": "11.5",
+                "version_removed": "15"
+              }
+            ],
+            "opera_android": [
+              {
+                "partial_implementation": true,
```

Comment:
Partial support here means on inline element, and prefixed with -webkit.

Reply:
Fixed.